From a5727754cd668f9a89338c6f299f36853493807c Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 17:55:53 +0700
Subject: [PATCH 693/909] Revert "arm64: dts: renesas: r8a7795-es1: Enable
 IPMMU devices"

commit 7b5cb132531c81ac6bc58246cab46abb674c7d09 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 4de1d0ffe6ee61de3f62302fc31b474316a2b832
to keep all IPMMU devices are disabled by default,
maintain the same IPMMU support environment with previous
released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 72d0c0fa7000..508a7ac1cef2 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -36,6 +36,7 @@
 		reg = <0 0xe7730000 0 0x1000>;
 		renesas,ipmmu-main = <&ipmmu_mm 8>;
 		#iommu-cells = <1>;
+		status = "disabled";
 	};
 
 	/delete-node/ usb-phy@ee0e0200;
-- 
2.17.1

